src/styledButton.tsx

```diff
@@ -0,0 +1,82 @@
+import { Interpolation, Theme } from "@emotion/react";
```

Comment:
Because `button.tsx` already exists. It contains the `ProtoButton` component. I am lacking in file name preferences, so suggest whatever you think is appropriate.

Reply:
Oh woops, I was blind apparently :D ehm but ye just put all this code into `button.tsx` as well. I don't see any reason why it shouldnt share a file with `ProtoButton`